test/k6/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": ["@babel/env", "@babel/typescript"],
+  "plugins": [
+    "lodash",
+    "@babel/proposal-class-properties",
+    "@babel/proposal-object-rest-spread"
```

Comment:
do we need this plugin even if we're not explicitly using the spread operator?

Reply:
not right now, but soon we will... for example

```
http.puthttps://foo.bar", {headers: {...defaultHeaders(), foo: 'bar'}})
```

Reply:
exactly, spread is a handy one 👍